client/tailwind.config.js

```diff
@@ -0,0 +1,32 @@
+module.exports = {
```

Comment:
Even though you're using purgecss, you might want to look at the final stylesheet. Tailwind notoriously creates giant sizes, because by default everything is on. Either purgecss is doing a lot of heavy lifting in the post processing, because their probably is a lot of unused stylesheets. Regards, I recommend whitelisting what you need from tailwind so that it is not generating larger files than you need it to.

Reply:
Can you say more about this? Do you mean that I shouldn't use `extend` but instead should overwrite and only define styles I need?

Reply:
More about explicitly stating tailwind plugins and variants and the issues with tailwind file sizes. https://tailwindcss.com/docs/controlling-file-size/